tendrl/node_agent/manager/rpc.py

```diff
@@ -0,0 +1,111 @@
+import json
+import logging
+import re
+import traceback
+import uuid
+
+import etcd
+import gevent.event
+
+from tendrl.node_agent.config import TendrlConfig
+
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+
+
+class EtcdRPC(object):
+
+    def __init__(self):
+        etcd_kwargs = {'port': int(config.get("bridge_common", "etcd_port")),
+                       'host': config.get("bridge_common", "etcd_connection")}
+
+        self.client = etcd.Client(**etcd_kwargs)
+        self.bridge_id = str(uuid.uuid4())
+
+    def _acceptor(self):
+        while True:
+            jobs = self.client.read("/api_job_queue")
+            for job in jobs.children:
+                raw_job = json.loads(job.value.decode('utf-8'))
+                # Pick up the "new" job that is not locked by any other bridge
+                if raw_job['status'] == "new":
```

Comment:
As discussed we should have some mechanism to figure out if the job is actually meant for the node agent and not for the storage bridge here. Just based on status of the job, there could be mix of storage bridge job and node agent jobs being picked here for processing.